build-system/tasks/integration.js

```diff
@@ -68,6 +68,9 @@ integration.flags = {
   'config':
     '  Sets the runtime\'s AMP_CONFIG to one of "prod" (default) or "canary"',
   'coverage': '  Run tests in code coverage mode',
+  'allow_debugger':
```

Comment:
For simplicity, how about renaming this `--debug`? (Assuming it doesn't clash with other known flags.)

Reply:
done